README.md

```diff
@@ -13,8 +13,28 @@ The compiler repo can be found [here](https://github.com/softdevteam/ykrustc).
 To get started, see
 [this section in the documentation](https://softdevteam.github.io/ykdocs/dev/getting_started.html).
 
-This repo uses xtask instead of regular cargo, so instead of running `cargo
-<target>`, you will instead need to execute `cargo xtask <target>`.
+This repo contains two separately-compiled workspaces, and thus the build
+system is a little non-standard. To act recursively on both workspaces, instead
```

Comment:
All this talk about "recursion" seems to me like an implementation detail that isn't of much importance to whoever indents to use/build this. Should we simplify these descriptions by removing any mention to recursion and just say something like: "To build both workspaces, use `cargo xtask target` instead of `cargo target`."

Reply:
I did start with something like that, but I thought people would ask "so how comes it works when I consume yk? Dependencies can't run xtask!".

Reply:
I don't know why anyone would ask this. The README says this is how you build it so that's what you have to do. ;-) And if they are still interested they can go read the code and its comment. My worry is that it's more likely to confuse people reading the README about what any of this means and why they should know about it, if all they want to do is build the thing.